### ISSUE TITLE
Disable browser completely

### ISSUE THREAD

**Issue by orbatschow** [burocratik/outdated-browser#303]
Is it possible to disable specific browsers entirely (e.g. EDGE) from showing up in the list?

**Comment by i2chris** [comment]
Yes, that would be great.  My website just about supports IE11, and I won't complain if you use it, but I don't want people installing it when they go to this website.
